### PR TITLE
Added ProductCategory to OJP

### PR DESCRIPTION
**Pull request by ue71603**
Only delivery

### CODE REVIEW

File: OJP_JourneySupport.xsd [hunk 1]
```diff
@@ -21,6 +21,23 @@
 			</xs:element>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:group name="ProductCategoryGroup">
+		<xs:annotation>
+			<xs:documentation>A simple form of ProductCategory from NeTExt</xs:documentation>
```
Comment (skinkie) [inline OJP_JourneySupport.xsd:26]: NeTExt -> NeTEx

**Comment by ue71603** [comment]
error in line 26: for ProductCategory

**Comment by ue71603** [comment]
Do not use capitals  for PRODUCT CATEGORY

**Comment by ue71603** [comment]
Replaced with new pull request

**Pull request closed without merging**